todo_app/android/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Mon Oct 02 11:45:24 CEST 2023
```

Comment:
A .gradle mappa is egy automatikusan generált mappa. Ezt is vegyük fel a gitignore-ba kérlek és ne pusholjuk. Automatikusan generált fileokat azzért nem szoktunk felpusholni, mert azt mindenkinek legenerálja a saját fejlesztőkörnyezete, így feleslegesen foglalják csak a helyet.